website/server/middlewares/cron.js

```diff
@@ -6,9 +6,10 @@ import Bluebird from 'bluebird';
 import { model as Group } from '../models/group';
 import { model as User } from '../models/user';
 import { recoverCron, cron } from '../libs/cron';
-import { v4 as uuid } from 'uuid';
 
 const daysSince = common.daysSince;
+// Wait this length of time in ms before attempting another cron
+const cronTimeout = new Date(3600000).getTime();
```

Comment:
I suggest that the comment should say something like just `// milliseconds` with no value specified (the existing comment might be enough). If the comment says specifically `1 h` there's a good chance we'll forget to update it if we change the value, and comments that get out of sync with the code are evil incarnate. :)